### PR TITLE
editorial: add example of .toJSON()+fetch POST

### LINKED ISSUE #576: Need short example of serializing response
Be nice to have a short example of `response.toJSON()` and `POST`ing it with `fetch()` to a server. That would "tell the complete story" of how to use the API. 

We can slot this into the existing example for payment response.

### PR DESCRIPTION
**Pull request by marcoscaceres**
* closes #576

***
[Preview](https://s3.amazonaws.com/pr-preview/w3c/payment-request/toJSON_example2.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/payment-request/52563ad...4e34712.html)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -430,6 +430,38 @@
...
+        </p>
+        <pre class="example">
+          async function doPaymentRequest() {
+            const payRequest = new PaymentRequest(methodData, details, options);
+            const payResponse = await payRequest.show();
+            let result = "unknown";
+            try {
+              const httpResponse = await fetch("/process-payment", {
+                method: "POST",
+                headers: new Headers({ "Content-Type": "application/json" }),
```
Comment (domenic) [inline index.html]: `new Headers` is not necessary; you can pass the object literal directly.

File: index.html [hunk 2]
```diff
@@ -430,6 +430,38 @@
...
+                body: payResponse.toJSON(),
+              });
+              result = httpResponse.ok ? "success" : "fail";
+            } catch (err) {
+              console.error(err);
```
Comment (domenic) [inline index.html:458]: Should this set result = "fail"?

**Review by domenic — APPROVED** [review]
LGTM with potential improvements

File: index.html (hunk 2 above)
Comment (marcoscaceres) [reply index.html:458]: Probably... but wanted to show the "unknown" state somehow.

Comment (domenic) [reply index.html:458]: Maybe do a quick parse of response.json() ? I dunno. I have a hard time understanding "unknown" in real-world systems...

Comment (marcoscaceres) [reply index.html:458]: yeah, me too :( Ok, will just add a comment about "unknown" that it's an option that people can use if they need it... however, we should try to come up with something real or we should mark it "at risk".

Chrome currently does nothing with "unknown" (behaves the same as "success", in that no error is shown to the user). I don't know if we (Firefox) will do anything with "unknown" at this point.

Comment (marcoscaceres) [reply index.html:458]: Filed https://github.com/w3c/payment-request/pull/583

Comment (marcoscaceres) [reply index.html:458]: Removed mention of "unknown", as I have a feeling it's it won't survive CR.

**Pull request merged**